src/%IPM/Main.cls

```diff
@@ -581,6 +581,28 @@ generate /my/path -export 00000,PacketName2,IgnorePacket2^00000,PacketName3,Igno
 <parameter name="name" description="Name namespace, * or context name*" />
 </command>
 
+	<command name="enable">
+		<summary>
+			Enable IPM in other namespaces.
+		</summary>
+		<!-- Modifiers -->
+		<modifier name="version" aliases="v" value="true" description="Specific version of IPM to be installed. Must specify if quiet flag is set."/>
```

Comment:
Nit: Perhaps can mention that a special version of `latest` can be provided. Also, adjust the text to indicate that if not specified, the latest version from the registry will be installed (hence is not required if quiet flag is set)